ITR/data/base_providers.py

```diff
@@ -669,18 +776,17 @@ def _normalize_scope_targets(self, scope_targets):
                 target.netzero_year = netzero_year
         return unique_scope_targets
 
-    def project_ei_targets(self, company: ICompanyData, production_bm: pd.Series) -> ICompanyEIProjectionsScopes:
+    def project_ei_targets(self, company: ICompanyData, production_proj: pd.Series) -> ICompanyEIProjectionsScopes:
```

Comment:
this method is refactored significantly.
may i suggest to cover with UT, to avoid regressions in multiple conditional branches?